src/api_server/Cargo.toml

```diff
@@ -25,12 +26,14 @@ attestation-service = { git = "https://github.com/confidential-containers/attest
 base64.workspace = true
 cfg-if = "1.0.0"
 env_logger.workspace = true
+jsonwebtoken = {version = "8", default-features = false, optional = true }
```

Comment:
Only for note: we might need to think about unifying jwt libs, as we are using `jwt-simple` now for jwt verification. https://github.com/confidential-containers/kbs/blob/main/src/api_server/Cargo.toml#L28

But it is not related to this PR. We could start this in future

Reply:
That's a really good point, and I'd prefer if that PR would not introduce a new JWT crate: Either use the existing one or switch to a new, single one. @Lu-Biao Could we use `jwt-simple` here?

Reply:
'jwt-sample' and 'jsonwebtoken'  provide similar functions, such as creating a key object  and then verify the ‘jwt’, but ‘jsonwebtoke’ provides a function to create key object  from ‘JwkSet’ while ‘jwt-sample’ does not, and it is required to deserialize amber-certs that is a JwkSet. So I prefer switch to 'jsonwebtoken'